sympy/series/formal.py

```diff
@@ -940,7 +941,17 @@ class FormalPowerSeries(SeriesBase):
     """
     def __new__(cls, *args):
         args = map(sympify, args)
-        return Expr.__new__(cls, *args)
+        obj = Expr.__new__(cls, *args)
+
+        ak = list(args)[4][0]
+        k = ak.variables[0]
+        obj.ak_seq = sequence(ak.formula, (k, 1, oo))
+        obj.fact_seq = sequence(factorial(k), (k, 1, oo))
+        obj.bell_coeff_seq = obj.ak_seq * obj.fact_seq
+
+        k = Dummy('k')
+        obj.sign_seq = sequence((-1, 1), (k, 1, oo))
+        return obj
```

Comment:
I would suggest to move this logic inside ``__init__`` method instead.  This is an initialization step. I think you can do this in ``__init__``.